docs/reference/builder.md

```diff
@@ -311,6 +311,12 @@ commands using a base image that does not contain `/bin/sh`.
 > If you want shell processing then either use the *shell* form or execute 
 > a shell directly, for example: `RUN [ "sh", "-c", "echo", "$HOME" ]`.
 
+> **Note**:
+> If you choose to use the *shell* form, any time you want to continue a single
```

Comment:
I'm good with just adding it to the `RUN` section now, I suspect it's far more likely people will attempt to do this with `RUN` than on other instructions.